src/components/UserInputBlock.tsx

```diff
@@ -86,34 +155,56 @@ export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent,
       < div className='my-3 os-feedback-message ' ref={contentRefCallback} dangerouslySetInnerHTML={{ __html: ack }} />
       )
 
-  return (
-    <EventControlledContent waitForEvent={waitForEvent}>
+  if (interactiveState !== null && persistorStatus === PersistorStatus.Success) {
```

Comment:
This is subjective / cosmetic, but what do you think about having the comment path be the default return at the end and other two conditional on state of the persistor `.get()`? Since this is the most complicated return, it would also reduce some indentation on it :man_shrugging: .